src/components/subscriptions/TabContentTable.jsx

```diff
@@ -131,49 +117,84 @@ function TabContentTable({ enterpriseSlug }) {
       ))}
       {!isLoading && !errors && (
         <>
-          {tableData?.length > 0 ? (
-            <>
-              {hasNoRevocationsRemaining && (
-                <StatusAlert
-                  alertType="warning"
-                  message={(
-                    <>
-                      You have reached your revoke access limit. For help
-                      managing your subscription licenses,
-                      {' '}
-                      <Link to={`/${enterpriseSlug}/admin/support`} className="alert-link">
-                        contact Customer Support
-                      </Link>.
-                    </>
-                  )}
-                />
-              )}
-              <div className="table-responsive">
-                <Table
-                  data={tableData}
-                  columns={columns}
-                  className="table-striped"
-                />
-              </div>
-              <div className="mt-3 d-flex justify-content-center">
-                <Pagination
-                  onPageSelect={page => fetchSubscriptionUsers({ searchQuery, page })}
-                  pageCount={users.numPages}
-                  currentPage={currentPage}
-                  paginationLabel={activeTabData.paginationLabel}
-                />
+          {activeTab === TAB_ALL_USERS && tableData?.length === 0 ? (
+            <div className="container py-4 col-md-5 border shadow">
```

Comment:
I'm not sure `.container` is needed here. Also, fwiw, with Bootstrap, you typically would nest a column within a row.

This example uses a centered column in a Bootstrap row:

```html
<div class="row">
    <div class="col-6 offset-md-3">
        This is horizontally centered.
    </div>
</div>
```

Also, Paragon now exports some layout components from react-bootstrap for rows and columns, too if thats more convenient:

```jsx
<Row>
  <Col xs={12} md={{ span: 6, offset: 3 }}>
    This is horizontally centered.
  </Col>
</Row>
```

As is, you have a container that is also being treated as a column, and then columns nested inside of columns without a parent row in between first.